cmd/cli.go

```diff
@@ -23,6 +30,56 @@ func newCli(GitCommit string, GitDate string) *cli.App {
 				Description: "Display the current mocktimism config",
 				Action:      actionConfig,
 			},
+			{
+				Name:        "anvil",
```

Comment:
Note I'm adding individual service commands during development but the actual release will not have these. These are here mostly to help with development now